blog/admin.py

```diff
@@ -1,7 +1,58 @@
+from blog.models import Blog, Comment, UserVote, Profile
```

Comment:
local imports should be after the third party import.

Use isort for ordering the imports automatically. You will have to create a `Makefile` from where you can run your `make quality` command to run the quality.

Reply:
Imports have been sorted.
Added this to the makefile.